core/src/main/java/com/linecorp/armeria/common/stream/AbstractStreamMessageDuplicator.java

```diff
@@ -407,9 +467,7 @@ private void doCleanup() {
                 completionFutures.add(future);
             });
             downstreamSubscriptions.clear();
-            final CompletableFuture<Void> allDoneFuture = CompletableFuture.allOf(
-                    completionFutures.toArray(EMPTY_FUTURES));
-            allDoneFuture.handle((unused1, unused2) -> {
+            CompletableFutures.successfulAsList(completionFutures, cause -> null).handle((unused1, unused2) -> {
```

Comment:
If we don't need results, I think `CompletableFutures.allAsList` is faster than `CompletableFutures.successfulAsList`.
Because of the call stack `successfulAsList` -> `joinList` -> `allAsList` :-) 

https://github.com/spotify/completable-futures/blob/06af851211d335de98fa501f0eb26086b6520888/src/main/java/com/spotify/futures/CompletableFutures.java#L101-L107

Reply:
Sorry, I had a wrong review. I realized that `allAsList` doesn't wait for all responses if any of the responses has an exception. 
Thanks for the nice work!